src/plugins/auto/src/plugin.cpp

```diff
@@ -458,8 +458,44 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model_impl(const std::string
     if (model_path.empty()) {
         support_devices = filter_device_by_model(support_devices_by_property, model, load_config);
     } else {
-        // AUTO / MULTI don't support caching explicitly, but can redirect this functionality to actual HW plugin
```

Comment:
Why this part cannot be in core?

The core already keeps cache configuration per device could also store the computed hashes to identify the blobs instead AUTO?